File: 3.0/python/1.py
f = open("input.txt")
dict = {}
mx = 0

for line in f:
    b = line.split()
    line = "".join(b)
    for c in line:
        if c not in dict:
            dict[c] = 0
        dict[c] += 1
        mx = max(mx, dict[c])

s = ""
for i in dict:
    s += i
s = sorted(s)

for i in range(mx):
    str = ""
    for i in s:
        if dict[i] >= mx:
            str += "#"
        else:
            str += " "
    print(str)
    mx -= 1

print(*s, sep="")

File: 3.0/golang/1.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strings"
)

func main() {
	f, _ := os.Open("input.txt")
	defer f.Close()

	in := bufio.NewReader(f)

	mp := make(map[rune]int)
	mx := 1

	for {
		lineb, _, err := in.ReadLine()
		if err != nil {
			break
		}
		words := strings.Split(string(lineb), " ")
		line := strings.Join(words, "")

		for _, c := range line {
			if val, ok := mp[c]; ok {
				mp[c] = val + 1
				if mp[c] > mx {
					mx = mp[c]
				}
			} else {
				mp[c] = 1
			}
		}
	}

	var s []string
	for i := range mp {
		s = append(s, string(i))
	}

	sort.Strings(s)
	for i := mx; i > 0; i-- {
		str := ""
		for _, v := range s {
			if mp[[]rune(v)[0]] >= i {
				str += "#"
			} else {
				str += " "
			}
		}
		fmt.Println(str)
	}
	fmt.Println(strings.Join(s, ""))
}
